pkg/agent/route/route_linux.go

```diff
@@ -1245,6 +1258,38 @@ func (c *Client) initWireguard() {
 	}
 }
 
+func (c *Client) initNodeLatency() {
+	gateway := "antrea-gw0"
+	if c.networkConfig.TrafficEncapMode.String() == "networkPolicyOnly" {
```

Comment:
use the `IsNetworkPolicyOnly()` method on `c.networkConfig.TrafficEncapMode`

I think there are examples in this file